src/app/components/header/header.component.html

```diff
@@ -0,0 +1,26 @@
+<header class="c-header fx fx-xcenter">
```

Comment:
What means all those prefixes?

Reply:
fx is just flex, I can change it to full name
c-header is a BOM terminology